### PR TITLE
Added PHPUnit plugin options coverage

### LINKED ISSUE #299: Add PHPUnit plugin options "allowed_[classes|methods|lines]_coverage"
## Expected behavior

PHPunit plugin should fail when coverage is lower that `allowed_[classes|methods|lines]_coverage` option value. Config example:

```yml
test:
    php_unit:
        config:
            - "path/to/phpunit.xml"
        coverage: true
        allowed_lines_coverage: 60 
```

Also, according to #263 we should have ability to compare coverage with previous build. Config example:

```yml
test:
    php_unit:
        config:
            - "path/to/phpunit.xml"
        coverage: true
        allowed_methods_coverage: previous
        allowed_lines_coverage: previous - 0.1
```

### PR DESCRIPTION
**Pull request by slavkluev**
## Contribution type

*Feature - resolves #299 *

## Description of change

- added PHPUnit plugin options coverage
- NOT ADDED comparison coverage with previous build

### CODE REVIEW

**Comment by corpsee** [comment]
@slavkluev Good job! I will try to review PR as soon as posible.

File: src/Plugin/PhpUnit.php [hunk 1]
```diff
@@ -219,11 +215,55 @@ protected function runConfig($directory, $configFile, $logFormat)
...
+     *
+     * @param array $coverage
+     *
+     * @return bool
+     */
+    protected function checkRequiredCoverage($coverage)
+    {
+        foreach ($coverage as $key => $currentValue) {
+            if ($allowedValue = $this->options->getOption(implode('_', ['required', $key, 'coverage']))) {
+                if (floatval($currentValue) < floatval($allowedValue)) {
```
Comment (corpsee) [inline src/Plugin/PhpUnit.php]: @slavkluev You should use `bccomp` for comparison of floats. Could you improve it please?

File: src/Plugin/PhpUnit.php [hunk 2]
```diff
@@ -58,11 +58,14 @@ public static function pluginName()
...
     /**
      * Standard Constructor
-     * $options['config']    Path to a PHPUnit XML configuration file.
-     * $options['run_from']  The directory where the phpunit command will run from when using 'config'.
-     * $options['coverage']  Value for the --coverage-html command line flag.
-     * $options['directory'] Optional directory or list of directories to run PHPUnit on.
-     * $options['args']      Command line args (in string format) to pass to PHP Unit
+     * $options['config']                       Path to a PHPUnit XML configuration file.
+     * $options['run_from']                     The directory where the phpunit command will run from when using 'config'.
+     * $options['coverage']                     Value for the --coverage-html command line flag.
```
Comment (corpsee) [inline src/Plugin/PhpUnit.php]: Could you add new parameters to plugin documentation (https://github.com/php-censor/php-censor/blob/master/docs/en/plugins/php_unit.md) please?

File: src/Plugin/PhpUnit.php (hunk 1 above)
Comment (slavkluev) [reply src/Plugin/PhpUnit.php]: Sure. It's a good point.

File: src/Plugin/PhpUnit.php (hunk 2 above)
Comment (slavkluev) [reply src/Plugin/PhpUnit.php]: Not a problem.

**Comment by corpsee** [comment]
@slavkluev :+1:

**Pull request merged**